draft-irtf-cfrg-opaque.md

```diff
@@ -1174,6 +1148,9 @@ The OPAQUE AKE protocols are those which are specified in {{instantiations}}.
 Future specifications (such as {{I-D.sullivan-tls-opaque}}) MAY introduce other
 AKE instantiations.
 
+The EnvelopeMode value is defined in {{data-types}} and must also be agreed upon
```

Comment:
This implies (to me) some sort of negotiation. Since the EnvelopeMode is already in the tuple above, I'd drop that bit:

```suggestion
The EnvelopeMode value is defined in {{data-types}}.
```